frontend/components/sections/BannersSection/SingleBanner.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Button, Text } from '@chakra-ui/react';
+import { ResponsiveImage, Wrapper } from '@ifixit/ui';
+import type { Banner } from '@models/components/banner';
+import NextLink from 'next/link';
+import { SectionDescription } from '../SectionDescription';
+import { SectionHeading } from '../SectionHeading';
+
+interface SingleBannerProps {
+   banner: Banner;
+}
+
+export function SingleBanner({ banner }: SingleBannerProps) {
+   return (
+      <Box as="section" position="relative" w="full" pt="36" pb="16">
+         <Box
+            position="absolute"
+            bgGradient="linear(to-r, blackAlpha.600 50%, blackAlpha.400)"
+            zIndex={-1}
+            top="0"
+            left="0"
+            bottom="0"
+            right="0"
+         />
+         {banner.image && (
+            <Box
+               position="absolute"
+               zIndex={-2}
+               top="0"
+               left="0"
+               bottom="0"
+               right="0"
+            >
+               <ResponsiveImage
+                  src={banner.image.url}
+                  alt=""
```

Comment:
Should we use the image alt text from strapi too?

```suggestion
                  alt={banner.image.alternativeText ?? "store hero image"}
```

Reply:
I put an empty string here because I consider this to be a decorative background image carrying no useful information for assistive technology (https://www.w3.org/WAI/tutorials/images/decorative/). Do you think it's worth that the screen reader sees this background image?

Reply:
TIL that `alt=""` is best practice for decorative images. With that in mind, I agree with you that the screen reader does not need to see this image, since the associated banner text will be more descriptive.